src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -59,7 +59,7 @@ public function load($file, $type = null)
         }
 
         try {
-            $parsedConfig = $this->yamlParser->parse(file_get_contents($path), Yaml::PARSE_KEYS_AS_STRINGS);
+            $parsedConfig = $this->yamlParser->parse(file_get_contents($path));
```

Comment:
Not more than when we added it in 3.3 (for instance `!str foo` in routing files was parsed as `"foo"` in 3.2 and is as `"!str foo"` since 3.3). Reverting this change is in fact more intuitive imo as it does make the parser always spec compliant.